tests/components/upnp/test_config_flow.py

```diff
@@ -29,12 +29,18 @@
 
 from tests.common import MockConfigEntry, async_fire_time_changed
 
+pytestmark = [
+    pytest.mark.usefixtures("mock_ssdp"),
+    pytest.mark.usefixtures("mock_get_source_ip"),
+    pytest.mark.usefixtures("mock_upnp_device"),
```

Comment:
What's the reason we change the scope of when `mock_upnp_device` is applied? It used to be applied to all upnp tests.